README.md

```diff
@@ -1 +1,58 @@
 # book-store-react
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is a website that allows users to add/remove books from a list. This project is build using JavaScript objects and arrays.
+
+
+## Built With
+
+- HTML
+- CSS 
+- Javascript
+- React
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone repository in your local machine 
+- cd Portfolio
+- open `index.html` in your browser.
+- You can use live server.
+
+## Live Demo
+
+[Books Live](https://basirking.github.io/Awesome-Book/)
+
```

Comment:
- Please, kindly remove or update the live demo link as it takes users to another project which is not a good user experience. please, kindly fix this.